src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import React from 'react';
+import { filterLinks } from '../../helper/filterLinks';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filter: Filter,
+  onFilter: (filter: Filter) => void,
+  completedTodos: Todo[],
+  activeTodosCount: number,
+  deleteCompleted: (todos: Todo[]) => void,
+};
+
+export const Footer: React.FC<Props> = React.memo(
+  ({
+    filter,
+    onFilter,
+    completedTodos,
+    activeTodosCount,
+    deleteCompleted,
+  }) => {
+    return (
+      <footer className="todoapp__footer">
+        <span className="todo-count">
+          {`${activeTodosCount} items left`}
+        </span>
+
+        <nav className="filter">
+          {filterLinks.map(link => (
+            <a
```

Comment:
Using an anchor tag `<a>` with an onClick handler may not be the best option here. Consider using a `<button>` instead, as it's more semantically correct and better for accessibility 🚀

Reply:
still not fixed